bevy_renet/src/lib.rs

```diff
@@ -70,11 +121,32 @@ impl RenetClientPlugin {
         }
     }
 
-    pub fn send_packets_system(mut client: ResMut<RenetClient>, mut renet_error: EventWriter<RenetError>) {
+    pub fn send_packets(mut client: ResMut<RenetClient>, mut renet_error: EventWriter<RenetError>) {
```

Comment:
Using `_system` is a convention suggested by `@cart`. Also other systems still have `_system` suffix, so I would keep it.